src/components/switch/switch.scss

```diff
@@ -18,11 +18,12 @@
 }
 
 label {
-    line-height: normal;
-
+    @include lime-typography.typography(body2);
```

Comment:
I replaced your fixup with this.

`lime-typography` is just `mdc-typography`, but with our preferred defaults already configured.

Reply:
Same goes for `lime-theme` by the way; it's `mdc-theme` with our config applied.

https://github.com/Lundalogik/lime-elements/blob/main/src/style/internal/lime-theme.scss
https://github.com/Lundalogik/lime-elements/blob/main/src/style/internal/lime-typography.scss